pages/about.html

```diff
@@ -2,11 +2,43 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta type="author" content="Rae Elwell">
+  <meta type="description" content="Rae's Corner">
+  <title>About Me</title>
+  <link rel="stylesheet" href="/style.css">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
 </head>
 <body>
-  
+  <nav class="nav-block">
+    <div class="intro">
+      <h1>About Me</h1>
+  </div>
+    <ul class="main-nav">
+      <li><a href="index.html">Index</a></li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+      <li><a href="https://twitter.com/Raelet">Twitter</a></li>
+    </ul>
+    <div class="git-link">
+      <button onclick="window.location.href='https://github.com/raeelwell';"">GitHub</button>
+    </div>
```

Comment:
While the button is cool it's probably overkill to do an `onclick`:

You can do something similar by just wrapping the button in a link:

```suggestion
    <a href="https://github.com/raeelwell" class="git-link">
      <button>GitHub</button>
    </a>
```